timeline/views_ajax.py

```diff
@@ -12,6 +13,7 @@
 
 from timeline.models import ProjectEvent
 from timeline.templatetags.timeline_tags import get_status_style
+from timeline.templatetags.timeline_tags import get_event_extra_data, collect_extra_data
```

Comment:
If there are methods we now use only in client side rendering instead of server side template tags, make sure to move them away from the template tag module under an appropriate view or mixin.